pom.xml

```diff
@@ -86,7 +86,7 @@
 		<dependency>
 		    <groupId>org.apache.httpcomponents</groupId>
 		    <artifactId>httpclient</artifactId>
-		    <version>4.5.5</version>
+		    <version>4.5.3</version>
```

Comment:
Why do the version downgrade?

Reply:
Sorry for that, I can't figure out why the latest version doesn't work on my workstation.